x-pack/plugins/event_log/server/es/context.ts

```diff
@@ -12,14 +12,17 @@ import { initializeEs } from './init';
 import { ClusterClientAdapter, IClusterClientAdapter } from './cluster_client_adapter';
 import { createReadySignal, ReadySignal } from '../lib/ready_signal';
 
+export const RETRY_DELAY = 2000;
+
 export interface EsContext {
-  logger: Logger;
-  esNames: EsNames;
-  esAdapter: IClusterClientAdapter;
+  readonly logger: Logger;
```

Comment:
This is just some clean-up I noticed while editing the file.   This structure is not used outside of the event log, but the fields should be read-only as they are populated via a function call which creates an object that returns this interface.  Figured it would be helpful to make sure we don't _accidentally_ update these fields in our code ...

Same with `initialized` a few lines below, and the new `retryDelay` uses the same pattern.